samples/custom-consent-authorizationserver/samples-custom-consent-authorizationserver.gradle

```diff
@@ -1,12 +1,33 @@
-apply plugin: 'io.spring.convention.spring-sample-boot'
+plugins {
+	id "org.springframework.boot" version "2.5.12"
+	id "io.spring.dependency-management" version "1.0.11.RELEASE"
+	id "java"
+}
+
+group = project.rootProject.group
+version = project.rootProject.version
+sourceCompatibility = "11"
+
+repositories {
+	mavenCentral()
+}
 
 dependencies {
-	compile 'org.springframework.boot:spring-boot-starter-web'
-	compile 'org.springframework.boot:spring-boot-starter-thymeleaf'
-	compile 'org.springframework.boot:spring-boot-starter-security'
-	compile project(':spring-security-oauth2-authorization-server')
-
-	testCompile 'org.springframework.boot:spring-boot-starter-test'
-	testCompile 'org.springframework.security:spring-security-test'
-	testCompile 'net.sourceforge.htmlunit:htmlunit'
+	implementation "org.springframework.boot:spring-boot-starter-web"
+	implementation "org.springframework.boot:spring-boot-starter-thymeleaf"
+	implementation "org.springframework.boot:spring-boot-starter-security"
+	implementation "org.springframework.boot:spring-boot-starter-jdbc"
+	implementation project(":spring-security-oauth2-authorization-server")
+	runtimeOnly "com.h2database:h2"
+
+	testImplementation "org.springframework.boot:spring-boot-starter-test"
+	testImplementation "org.junit.jupiter:junit-jupiter"
```

Comment:
It seems to be. There must have been something in the `io.spring.convention.spring-sample-boot` plugin that was making JUnit 4 work previously. Since Spring Security isn't using it, I didn't investigate further. With just the `org.springframework.boot` plugin from start.spring.io, this is needed until we upgrade to JUnit 5.